### ISSUE TITLE
Shoulder external rotation limited

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10943]
**Preferred term label:**
Shoulder external rotation limited

**Synonyms**
Reduced shoulder range of motion; stiff shoulder

**Definition (free text, please give PubMed ID)**
Difficulty rotating the lower part of the arm away from the body (tested with the elbow flexed and the elbow held next to the side of the body) secondary to pain, weakness, or mechanical block

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Rotator cuff tear

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001194
- label: Limited shoulder external rotation
- parents: [Limited shoulder movement (HP:0006467)](https://hpo.jax.org/app/browse/term/HP:0006467)
- pmid: 
- def: Difficulty rotating the lower part of the arm away from the body (tested with the elbow flexed and the elbow held next to the side of the body) secondary to pain, weakness, or mechanical block.
- comment: Diseases characterized by this finding include rotator cuff tear.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10943
- synonyms: Shoulder external rotation limited(NONE)
